project/Publish.scala

```diff
@@ -40,19 +40,14 @@ object Publish extends AutoPlugin {
   }
 
   private def akkaPublishTo = Def.setting {
-    sonatypeRepo(version.value).orElse(localRepo(defaultPublishTo.value))
+    val key = new java.io.File(
+      Option(System.getProperty("akka.gustav.key", null)).getOrElse(sys.env("HOME") + "/.ssh/id_rsa_gustav.pem"))
```

Comment:
`null` is already the default for if property not defined

Reply:
`System.getProperty("user.home")` might be more platform neutral than env variable